src/common.js

```diff
@@ -16,11 +16,9 @@ let __nimiqLoaded = null;
 if ((BrowserDetection.isIOS() || BrowserDetection.isSafari()) && 'serviceWorker' in navigator) {
     // Register service worker to strip cookie from requests.
     // This file is always called from a ./request/*/ folder, hence the paths.
-    navigator.serviceWorker.register('../../ServiceWorker.js', {
-        scope: '../../',
-    }).then(reg => {
+    navigator.serviceWorker.register('/ServiceWorker.js').then(reg => {
```

Comment:
<img width="721" alt="Screen Shot 2019-05-21 at 13 10 52" src="https://user-images.githubusercontent.com/43245427/58091461-eb6ddf00-7bc9-11e9-8bfb-811ccd38d954.png">
I don't think that works as intended.

Reply:
```suggestion
    navigator.serviceWorker.register('/src/ServiceWorker.js').then(reg => {
```

Reply:
ah, it fails locally... :(

Reply:
Ok, I'll remove the registration adaptions